completions/Makefile.am

```
@@ -501,27 +500,37 @@ bashcomp_DATA = 2to3 \
 		zopfli \
 		zopflipng
 
+solaris_extra = pkgutil
 
-if SOLARIS
-bashcomp_DATA += pkgutil
-endif
-
-if BSD
-bashcomp_DATA += pkg_delete
-endif
+bsd_extra = pkg_delete
 
-if FREEBSD
-bashcomp_DATA += \
+freebsd_extra = \
 		freebsd-update \
 		kldload \
 		kldunload \
 		portinstall \
 		portsnap \
 		portupgrade
-endif
 
+# We include all the files in the tarball
+EXTRA_DIST = $(bashcomp_DATA) \
+	     $(solaris_extra) \
+	     $(bsd_extra) \
+	     $(freebsd_extra)
```

It doesn't cause a problem, but this causes duplicates because `*_extra` is included in `bashcomp_DATA` depending on the platform.

```suggestion
EXTRA_DIST = $(cross_platform) \
	     $(solaris_extra) \
	     $(bsd_extra) \
	     $(freebsd_extra)
```